code/modules/vore/appearance/sprite_accessories_taur_vr.dm

```diff
@@ -61,25 +66,44 @@
 		return FALSE
 	if(!ishuman(M))
 		return FALSE
-	if(M.size_multiplier > size_multiplier)
-		to_chat(M,"<span class='warning'>This isn't a pony show! They need to be bigger to ride.</span>")
+	if(M in buckled_mobs)
+		return FALSE
+	if(M.size_multiplier > size_multiplier * 1.2)
```

Comment:
Maaaybe. Their feet might also stick through the bottom of taur. Did you test with a 120 scale person on a 100 person?

Reply:
They get lifted up by 25% via the offsets.